Add new lobby

coolpeter98 opened the pull request:
This adds a lobby with giant spruce trees, caves, parkour and more. Relevant files:
[schematics.tar.gz](https://github.com/user-attachments/files/21605506/schematics.tar.gz)

coolpeter98 committed:
Add new lobby

coolpeter98 committed:
Fix cut off and make stuff look better

coolpeter98 committed:
Add parent folder so Rojo stops complaining

EliTheGingerCat commented on src/Workspace/Lobby.rbxm:
1. I would prefer not to use Minecraft names for the map instances. This is not visible to the player, so perhaps it does not matter, but it also means that it does not hurt the player if everything is simply named "Part" and "UnionOperation".
2. The map is far from the origin for some reason. This does not really matter, but I would prefer it to be centred on the origin ((0, 0, 0)).

Even though I am requesting changes, feel free to argue against these points.

coolpeter98 replied:
1. It is easier to edit the map this way because you can just search for a specific block and select all easily.
2. Ok, do I center the map or spawn location?

EliTheGingerCat replied:
1. Refuted.
2. Resolved. (I centred the map.)

Next time, I will make separate comments so that issues can be tracked easier.

EliTheGingerCat commented on src/Workspace/Lobby.rbxm:
Are there supposed to be [`Snaps`](<https://create.roblox.com/docs/reference/engine/classes/Snap>) everywhere?

coolpeter98 replied:
No.  Must be an issue with rojo or the converter, i don't remember adding those, and i had no reason to add them either.

EliTheGingerCat replied:
Yeah, it seems to be an issue with `rojo build` since manually dropping the RBXM into Roblox Studio results in a map that seemingly does not contain any Snaps.

EliTheGingerCat committed:
fix: centre lobby

Also turn the lobby from a Folder to a Model. This makes it easier to
move the lobby around.

EliTheGingerCat committed:
fix: snap map to cube grid
